src/Entity/PackingListInfo.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Ups\Entity;
+
+use DOMDocument;
+use Ups\NodeInterface;
+
+/**
+ * @author Thijs Wijnmaalen <thijs@wijnmaalen.name>
+ */
+class PackingListInfo implements NodeInterface
+{
+    /**
+     * @var array
+     */
+    private $packageAssociated = [];
+
+    /**
+     * @param string $number
+     * @param string $amount
+     *
+     * @return $this
+     */
+    public function addPackageAssociated($number, $amount)
+    {
+        $object = new \stdClass();
```

Comment:
Can you create a PackageAssociated entity and have that as an argument for this method? That entity can have a toNode method also again, so we keep the flow of letting the entity create it's node itself.